Makefile

```diff
@@ -12,19 +12,19 @@ bench: ## Benchmark the project
 build: ## Build the project
 	source test-utils.sh ;\
 	section "Cargo build" ;\
-	cargo build
+	cargo build --all
```

Comment:
We need the `--all` flag now to pick up `gosub-bindings`.  So perhaps good to use `make build` now.

Reply:
Maybe you could add a target to build only one specific. But it might make sense to do this as a seperate PR